src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -28,9 +28,12 @@
  * Contains utility methods used for parsing strings in the various *Parser classes.
  */
 public class ParserUtil {
+    public static final int MIN_DURATION = 10;
+    public static final int MAX_DURATION = 840;
```

Comment:
Do you think it would be good to add a comment on top to explain the numbers? or it's quite self-explanatory

Reply:
I agree with Jinhao about this point 😄